modules/API/external/external_api.js

```diff
@@ -84,323 +70,342 @@ function changeParticipantNumber(APIInstance, number) {
+     * @param {boolean} [noSSL] - If the value is true https won't be used.
+     * @param {string} [jwt] - The JWT token if needed by jitsi-meet for
+     * authentication.
+     */
+    constructor(domain, // eslint-disable-line max-params
+        roomName = '', width = MIN_WIDTH, height = MIN_HEIGHT,
+        parentNode = document.body, configOverwrite = {},
+        interfaceConfigOverwrite = {}, noSSL = false, jwt = undefined) {
+        super();
+        this.parentNode = parentNode;
+        this.url = generateURL(domain, {
+            configOverwrite,
+            interfaceConfigOverwrite,
+            jwt,
+            noSSL,
+            roomName
+        });
+        this._createIFrame(Math.max(height, MIN_HEIGHT),
+            Math.max(width, MIN_WIDTH));
+        this.postis = postisInit({
+            scope: `jitsi_meet_external_api_${id}`,
+            window: this.frame.contentWindow
+        });
+        this.numberOfParticipants = 1;
+        this._setupListeners();
+        id++;
     }
-    sendMessage(this.postis, {method: commands[name], params: argumentsList});
-};
 
-/**
- * Executes commands. The available commands are:
- * displayName - sets the display name of the local participant to the value
- * passed in the arguments array.
- * toggleAudio - mutes / unmutes audio. no arguments
- * toggleVideo - mutes / unmutes video. no arguments
- * filmStrip - hides / shows the film strip. no arguments
- * toggleChat - hides / shows chat. no arguments.
- * toggleContactList - hides / shows contact list. no arguments.
- * toggleShareScreen - starts / stops screen sharing. no arguments.
- * @param object the object with commands to be executed. The keys of the
- * object are the commands that will be executed and the values are the
- * arguments for the command.
- */
-JitsiMeetExternalAPI.prototype.executeCommands = function(object) {
-    for(var key in object)
-        this.executeCommand(key, object[key]);
-};
+    /**
+     * Creates the iframe element.
+     *
+     * @param {number} height - The height of the iframe.
+     * @param {number} width - The with of the iframe.
+     * @returns {void}
+     *
+     * @private
+     */
+    _createIFrame(height, width) {
+        this.iframeHolder
+            = this.parentNode.appendChild(document.createElement('div'));
+        this.iframeHolder.id = `jitsiConference${id}`;
+        this.iframeHolder.style.width = `${width}px`;
+        this.iframeHolder.style.height = `${height}px`;
+        this.frameName = `jitsiConferenceFrame${id}`;
+        this.frame = document.createElement('iframe');
+        this.frame.src = this.url;
+        this.frame.name = this.frameName;
+        this.frame.id = this.frameName;
+        this.frame.width = '100%';
+        this.frame.height = '100%';
+        this.frame.setAttribute('allowFullScreen', 'true');
+        this.frame = this.iframeHolder.appendChild(this.frame);
```

Comment:
Which lines? If you mean `this.frame.width = '100%'` this is old code!

I don't think this change fits anywhere in this commits and the PR. Also I'm not sure whether I'd implement it with `frame.style.border = 0;` (of course we can start with this and change it later). 

Also in this PR there aren't any visual changes.